src/gfx_device/gfx_device.rs

```diff
@@ -59,7 +59,7 @@ impl GfxDevice {
         // the scene.
         for (rend, entity) in (&renderables, &entities).iter() {
             let global_trans = match global_transforms.get(entity) {
-                Some(gt) => gt.clone(),
+                Some(gt) => *gt,
```

Comment:
Okay, thanks for checking! I don't have my computer in front of me at the moment, and I'm operating off my phone. 👍